.github/workflows/ci-pkg-install.yml

```diff
@@ -28,52 +31,71 @@ defaults:
 
 jobs:
 
+  build-packages:
+    uses: ./.github/workflows/_build-packages.yml
+    with:
+      artifact-name: dist-packages-${{ github.sha }}
+
   install-pkg:
+    needs: build-packages
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04, macOS-12, windows-2022]
         pkg-name: ["app", "lite", "pytorch", "lightning", "notset"]
         python-version: ["3.7" , "3.10"]
+        # TODO: add also install from source
     steps:
       - uses: actions/checkout@v3
-
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
-
-      - name: DocTests actions
-        working-directory: .actions/
-        run: |
-          pip install -q pytest
-          python -m pytest setup_tools.py
-
-      - run: python -c "print('NB_DIRS=' + str(2 if '${{ matrix.pkg-name }}' == 'pytorch' else 1))" >> $GITHUB_ENV
-
-      - uses: ./.github/actions/pkg-check
+      - uses: actions/download-artifact@v3
         with:
-          pkg-name: ${{ matrix.pkg-name }}
-          nb-dirs: ${{ env.NB_DIRS }}
+          name: dist-packages-${{ github.sha }}
+          path: dist
 
+      - run: |
+          python -c "print('PKG_DIR=' + {'notset': 'lightning'}.get('${{matrix.pkg-name}}', '${{matrix.pkg-name}}'))" >> $GITHUB_ENV
       - uses: ./.github/actions/pkg-install
         with:
+          pkg-folder: dist/${{ env.PKG_DIR }}
           pkg-name: ${{ matrix.pkg-name }}
 
       - name: Run CLI (via python)
         if: ${{ matrix.pkg-name == 'lightning' || matrix.pkg-name == 'notset' }}
         run: python -m lightning --version
-
       - name: Run CLI (direct bash)
         if: ${{ matrix.pkg-name == 'lightning' || matrix.pkg-name == 'app' }}
         run: lightning --version
 
+      - name: Adjust code for Lit
+        if: ${{ matrix.pkg-name == 'lightning' || matrix.pkg-name == 'notset' }}
+        run: |
+          pip install -q -r .actions/requirements.txt
+          python .actions/assistant.py copy_replace_imports --source_dir="./src" \
+            --source_import="pytorch_lightning,lightning_lite,lightning_app" \
+            --target_import="lightning.pytorch,lightning.lite,lightning.app"
+          rm -rf src/lightning
+      - name: Rename src folders
+        working-directory: src/
+        run: |
+          mv pytorch_lightning pl
+          mv lightning_lite lit_lite
+          mv lightning_app lit_app
```

Comment:
Can you explain why move these directories?

Reply:
so there is no accidental use of local version but importing the install version